packages/pn-personafisica-webapp/src/redux/notification/types.ts

```diff
@@ -0,0 +1,283 @@
+import { NotificationStatus } from "@pagopa-pn/pn-commons";
```

Comment:
@VincenzoLiberti spostare questi types nel file pn-commons/types/Notifications e rifattorizzare di conseguenza questo file (eliminandolo) e il file gemello in pn-pa